### ISSUE TITLE
Use without NPM

### ISSUE THREAD

**Issue by EliotHertenstein** [korywka/mapbox-gl-controls#31]
I've noticed that many plugins allow users to use an api.mapbox.com URL to import the plugin into an HTML file. Is this possible with gl-controls? If so, what is the URL?

**Comment by korywka** [comment]
Nope, it is not possible. For zero configuration you can use https://parceljs.org/

**Comment by PedroVenancio** [comment]
Hi @korywka 
It would be great to have gl-controls available to load via a CDN service, such as unpkg, using a script tag. 
I hope it can be done in near future.
Thanks! Best regards!

**Comment by korywka** [comment]
🚫 blocker mapbox issue: https://github.com/mapbox/mapbox-gl-js/issues/8676
